Database/Objects/Views/dbo.vViewTreatmentBMPModelingAttributes.sql

```diff
@@ -0,0 +1,42 @@
+Drop view if exists dbo.vViewTreatmentBMPModelingAttributes
+GO
+
+Create View dbo.vViewTreatmentBMPModelingAttributes

```

Comment:
I'm actually really glad that you decided to make a view for this b/c I think moving forward we're going to want most of our grids to be view-driven for performance reasons, so, awesome.